### ISSUE TITLE
Writing An Evaluation of Deliverables-Requirements 

### ISSUE THREAD

**Issue by mertcan-ozkan** [bounswe/bounswe2022group3#137]
### Task:
As stated in issue #132 :  Evaluation of requirements will be written as a subsection of Evaluation of Deliverables. This issue is closely related to [scenarios and mockups evaluation](https://github.com/bounswe/bounswe2022group3/issues/139) written by @ahmetkudu since they will  be put under the same deliverables subsection. Before writing, I will also check the old milestone 1 reports from previous semesters to understand better what should be written here .

### Place To Put The Documentation:
Milestone 1 Report (After its done there will be a link here)

### Deadline: 
14.04.2022 20.00

**Comment by mertcan-ozkan** [comment]
I have checked the evaluation of Deliverables of the following groups from their milestone 1 report:
2018 group [10](https://github.com/bounswe/bounswe2018group10/blob/develop/resources/CmpE%20451%20Fall%202018%20Milestone1.pdf) 
2019 group [5 ](https://github.com/bounswe/bounswe2019group5/blob/master/deliverables/M1_Group5.pdf)
2020 group [7](https://github.com/bounswe/bounswe2020group7/blob/master/deliverables/M1-Group7.pdf)
2021 group [3 ](https://github.com/bounswe/2021SpringGroup3/blob/master/deliverables/CMPE%20352%20Milestone%20Report1.pdf)
2021 group [9](https://github.com/bounswe/2021SpringGroup9/blob/master/deliverables/M1-Group9.pdf)
2021 group [5](https://github.com/bounswe/2021SpringGroup5/blob/master/deliverables/M1-Group5.pdf)
2021 group [7](https://github.com/bounswe/2021SpringGroup7/blob/master/deliverables/M1-Group7.pdf)
2021 group[12](https://github.com/bounswe/2021SpringGroup12/blob/master/deliverables/M1-Group12.pdf)

-It seems that majorty of the reports have just written a paragraph for the requirements evaluation. I will use these examples while writing the section, but I may give more details.

**Comment by mertcan-ozkan** [comment]
Evaluation of Requirements 
Initial draft for requirements were created at the very start of the project, based on the project description to capturre the general idea of the project. After the first customer meeting, a new version was created based on the feedback given by the customer. Throughout the project, we continued editing requirements based on feedback, also there were two major changes in the requirements. The first major change was made while creating the mockups and scenarios, where we needed to consider the user requirements thoroughly. The second major change was made while we were designing the use case, sequence and class diagrams, where we needed to specify how our project would work and interract with the users. After all these changes and improvements, the requirements are finally completed as a delivarable. Requirements are very vital to our project since it will be used to define the desired features ,with its boundaries, that our project will have to provide; along with the features that users will be using. During the creation of the very first draft; each group member was assigned to create the initial version of a subsection of requirements, then, all of initial sections were modfied and changed by the group members numerous times and these changes were also reviewed by the group members in the group meetings.
